camera.py

```diff
@@ -0,0 +1,57 @@
+import requests
+from requests.auth import HTTPDigestAuth
+
+class camera:
```

Comment:
In Python, class names usually start with uppercase letters. See the [naming convention in PEP8](https://peps.python.org/pep-0008/#class-names) for details.

```suggestion
class Camera:
```